src/core/modules/memory/memory_function.cpp

```diff
@@ -179,6 +179,29 @@ CFunction::CFunction(unsigned long ulAddr, Convention_t eCallingConvention,
 	m_oConverter = oConverter;
 }
 
+CFunction::CFunction(const CFunction& obj)
+	:CPointer(obj)
+{
+	m_tArgs = obj.m_tArgs;
+	m_eReturnType = obj.m_eReturnType;
+	m_oConverter = obj.m_oConverter;
+
+	m_eCallingConvention = obj.m_eCallingConvention;
+	m_iCallingConvention = obj.m_iCallingConvention;
+
+	if (m_eCallingConvention != CONV_CUSTOM)
+	{
+		m_pCallingConvention = MakeDynamicHooksConvention(m_eCallingConvention, ObjectToDataTypeVector(m_tArgs), m_eReturnType);
+		m_oCallingConvention = object();
+	}
+	else
+	{
+		m_pCallingConvention = obj.m_pCallingConvention;
+		m_oCallingConvention = obj.m_oCallingConvention;
+		Py_INCREF(m_oCallingConvention.ptr());
```

Comment:
I haven't tested, only hovered the changes, but is that necessary? I guess this is an additional reference reserved for DynamicHooks and it will be taken care of as well when the copy gets destroyed?

Reply:
This process is necessary to prevent double freeing in the destructor, but it seems to cause a memory leak when the function is hooked related to another bug.

Details will be provided in a new pull request.